### DISCUSSION TITLE
Sort Replay Titles on Library

### DISCUSSION THREAD

**Discussion by BwL1289** [RecordReplay/devtools#4857] [Ideas]
I've developed a custom feature implementation/bug flow around replay—specifically replay titles and how they tie into Jira. 

It would be very helpful to be able to sort the titles (beginning on prefix) on the Library page.

Happy to provide the workflow we've developed if it's helpful to anyone else.

**Comment by jonbell-lot23** [comment]
March 15, 2022:

Hello @BwL1289 ! Thanks for this reminder -- the library works ok for now, but we can definitely improve it further. Being able to sort would be a helpful improvement.
